### DISCUSSION TITLE
Flet on Raspberry (32 bit) [Question]

### DISCUSSION THREAD

**Discussion by pasqualepepe887** [flet-dev/flet#2978] [Q&A] [answered]
### Question

Hi,
there isn't a way to use Flet on an old Raspberry (pi 3 A+ model 32 bit). I love Flet but I am not able to work on raspberry.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment, marked as answer]
It should be 64-bit Raspberry. 32-bit is not supported.

**Reply by pasqualepepe887** [reply]
Thanks, I will use a 64 bit OS

**Comment by pasqualepepe887** [comment]
It should be 64-bit Raspberry. 32-bit is not supported.
